POLICIES.md

```diff
@@ -228,8 +231,106 @@ spec:
 
 **Note** Recently pushed images to the registry that have not completed scanning are denied admission.
+
+## Customizing policies
+
+You can change the policy that Portieris uses to permit images, either at the cluster or Kubernetes namespace level. In the policy, you can specify different enforcement rules for different images.
+
+**Tip** You must have a policy set. Otherwise, deployments to your cluster fail. If you don't want to enforce any image security policies, [Uninstall Portieris](https://github.com/IBM/portieris#uninstalling-portieris).
+
+When you apply a deployment, Portieris checks whether the Kubernetes namespace that you are deploying to has a policy to apply. If it does not, Portieris uses the cluster-wide policy. Your deployment is denied if no namespace or cluster-wide policy exists.
+
+The following table explains the `.yaml` components that you must set in your Kubernetes custom resource definition `.yaml` file.
+
+
+| Field | Description |
+|-------|-------------|
+| `kind` | For a cluster-wide policy, specify the `kind` as `ClusterImagePolicy`. For a Kubernetes namespace policy, specify as `ImagePolicy`. |
+|`metadata/name` | Name the custom resource definition. |
+| `spec/repositories/name` | Specify the repositories to allow images from. Wildcards (`*`) are allowed in repository names. Repositories are denied unless a matching entry in `repositories` allows it or applies further verification. An empty `repositories` list blocks deployment of all images. To allow all images without verification of any policies, set the name to `*` and omit the policy subsections. |
+| `../../../policy` | Complete the subsections for `trust` and `va` enforcement. If you omit the policy subsections, it is equivalent to specifying `enabled: false` for each. |
+| `../../../../trust/enabled` | Set as `true` to allow only images that are [signed for content trust](https://cloud.ibm.com/docs/Registry?topic=Registry-registry_trustedcontent) to be deployed. Set as `false` to ignore whether images are signed. |
+| `../../../../trust/signerSecrets/name` | If you want to allow only images that are signed by particular users, specify the Kubernetes secret with the signer name. Omit this field or leave it empty to verify that images are signed without enforcing particular signers. For more information, see [Specifying trusted content signers in custom policies](#specifying-trusted-content-signers-in-custom-policies). |
+| `../../../../va/enabled` | Set as `true` to allow only images that pass the [Vulnerability Advisor](https://cloud.ibm.com/docs/Registry?topic=va-va_index) scan. Set as `false` to ignore the Vulnerability Advisor scan. |
+
+**Table 1**. Understanding the `.yaml` components for the Kubernetes custom resource definition.
+
+Before you begin, [target your `kubectl` CLI](https://cloud.ibm.com/docs/containers?topic=containers-cs_cli_install#cs_cli_configure) to the cluster. Then, complete the following steps:
```

Comment:
This link should be deleted - those instructions aren't valid for users not on IKS.